enhancements/observability/enable-network-observability-on-day0.md

```diff
@@ -0,0 +1,279 @@
+
+### Non-Goals
+
+There are other proposals to make Network Observability more visible and prominent, such as displaying a panel that would describe the features of Network Observability and provide a button to install it.  However, this feature enhancement addresses [OCPSTRAT-2469](https://issues.redhat.com/browse/OCPSTRAT-2469) that explicitly calls for Network Observability to be up and running after install.
+
+Network Observability Operator manages the components, such as flowlog pipelines.  Therefore, there is no need to consider the lifecycle management, since that will not change.
+
+## Proposal
+
+There are three OpenShift repositories that this proposal changes.  They are [openshift/api](https://github.com/openshift/api), [openshift/cluster-network-operator](https://github.com/openshift/cluster-network-operator), and [openshift/install](https://github.com/openshift/installer).
+
+### Repository: openshift/api
+
+The openshift/api repository is a shared repository for defining the API.  This adds the `installNetworkObservability` field in the Network Custom Resource Definition (CRD) under the spec section.
+
+```yaml
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  installNetworkObservability: true
+```
+
+Listing 1: Network manifest
+
+If the value is true or doesn't exist, Network Observability is enabled.  If it is set to false, Network Observability is not enabled or to be precise, *nothing is done*.  It doesn’t remove Network Observability if it is set to false, hence the reason *not* to call it `enableNetworkObservability`.  To be clear, `installNetworkObservability` not only installs Network Observability, but it also creates the FlowCollector custom resource.
+
+### Repository: openshift/cluster-network-operator
+
+The actual enabling of Network Observability is done in the Cluster Network Operator (CNO).  The rationale is that we want the network observability feature to be part of networking.  This is as opposed to being part of the general observability or as a standalone entity.  Yet, there is still a separation at the lower level so that the two can be independently developed and released at different times, particularly for bug fixes.
+
+In CNO, it adds a new controller for observability and adds it to the manager.  The controller is a single Go file where the Reconciler is called initially and reads the state of the installNetworkObservability field.  If true, it does the following:
+
+1. Check if Network Observability Operator (NOO) is installed. If yes, exit.
+2. Create "openshift-netobserv-operator" namespace if it doesn't exist.
+3. Install NOO using OLM's OperatorGroup and Subscription.
+4. Wait for NOO to be ready and the OpenShift web console to be available.
+5. Create the "netobserv" namespace if it doesn't exist.
+6. Check if a FlowCollector instance exists. If yes, exit.
+7. Create a FlowCollector instance.
```

Comment:
Concretely what gets deployed in terms of pods and services? Regarding pods, what level of customization is offered in terms of resource requests/limits and scheduling (infrastructure nodes)?